compiler/daml-lf-verify/daml/conditionals/daml.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
I'm not 100% sure we need the `daml.yaml` files for the test examples. Can you please check if the tests still work without them and remove them if possible. Same further below.